templates/common/infra/bicep/core/ai/project.bicep

```diff
@@ -36,8 +36,6 @@ resource project 'Microsoft.MachineLearningServices/workspaces@2024-01-01-previe
     hbiWorkspace: false
     v1LegacyMode: false
     publicNetworkAccess: publicNetworkAccess
-    discoveryUrl: 'https://${location}.api.azureml.ms/discovery'
```

Comment:
Does it not need the discoveryUrl?

Reply:
No, it doesn't.  It is automatically created when omitted.